packages/shared/src/components/cards/v1/Card.tsx

```diff
@@ -44,7 +44,7 @@ export const CardContent = classed('div', 'flex flex-col mobileXL:flex-row');
 
 export const CardImage = classed(
   Image,
-  'rounded-12 max-h-[calc(100vw-34px)] mobileXL:aspect-auto mobileXL:max-h-40 mobileXL:w-40 mobileXXL:max-h-56 mobileXXL:w-56',
+  'rounded-12 max-h-[12.5rem] object-cover w-full h-auto mobileXL:aspect-auto mobileXL:max-h-40 mobileXL:w-40 mobileXXL:max-h-56 mobileXXL:w-56',
```

Comment:
Just to understand, is this what was causing the main issue the PR was raised for?

Reply:
Yes, first the `w-full` ensure the image is rendered end to end. Then sets a max height which I fixed since it the calculation is based on width, which doesn't make sense.